code/obj/machinery/recharger.dm

```diff
@@ -157,22 +157,24 @@ obj/machinery/recharger
 	// WHO DID THIS
 	// WHY DID YOU DO THIS
 	// die
+	last_process = TIME
```

Comment:
please move the last_process thing to the base machinery loop or loop(s) if you can! having to re-set that manually when you could just call your parents seems kinda bad.

Reply:
Okay, did that!